tekton-proposal.md

```diff
@@ -0,0 +1,73 @@
+# Knative Build and Tekton Pipelines Steering Committee proposal
+
+Jason Hall ([jasonhall@google.com](mailto:jasonhall@google.com))
+
+Feb 25 2019
+
+## Objective
+
+Propose a resolution to the ongoing confusion and FUD around the future of Knative Build with regard to the recently announced Tekton Pipelines change. This doc should be used as a forum for external comment and questions, to resolve any concerns before the Steering Committee decides whether to adopt the proposal.
+
+Please leave comments on this PR. The text of the proposal may change based on feedback, and the PR will retain the history of these edits.
+
+## Background
+
+Knative Build was conceived at the beginning of the Knative project alongside the other serverless-focused components (Serving and Eventing) to enable a "serverless development experience". Its goal was to enable users to specify their source to a Knative Service deployment and the steps to take to make it a container, and deploy that container, while interacting with a single custom resource.
+
+From the beginning, Build had bigger ambitions. Its [2018 roadmap](https://github.com/knative/build/blob/master/roadmap-2018.md) mentions features that aren't directly related to furthering serverless deployments (triggering, workflow, etc.). These aren't accelerators for serverless deployments, these are CI/CD features!
```

Comment:
Great that there were bigger ambitions that fell more in the realm of CI/CD, but we should not undersell the fact that Building a container is absolutely needed for this knative implementation of serverless features.

Reply:
Building a container as part of a serverless deployment is definitely very much still in scope for Knative Build, which I hope is clear from the sections below. If there's a way you think I can make that clearer, feel free to make a suggestion.

Reply:
@ImJasonH

You could add a two column table one for KnativeBuild and another for TekTonCICD

And have side by side the features, goals, not goals on each as bullets with the one that need more description with link to a section on the same document